.travis.yml

```diff
@@ -7,6 +7,16 @@ services:
 # a lightweight environment
 language: c
 
+env:
+    - DC=dmd1 DIST=trusty F=production
+    - DC=dmd1 DIST=trusty F=devel
+    - DC=dmd1 DIST=xenial F=production
+    - DC=dmd1 DIST=xenial F=devel
+    - DC=dmd-transitional DIST=trusty F=production
+    - DC=dmd-transitional DIST=trusty F=devel
+    - DC=dmd-transitional DIST=xenial F=production
+    - DC=dmd-transitional DIST=xenial F=devel
```

Comment:
This will be more complex to achieve, since we need to rethink the docker images for that. Right now we are using global system installations of `dmd`, so we can only install `dmd1`, `dmd-transitional` and `dmd` because they use different config files. If we want to use multiple versions for each, we should move to another scheme, probably providing the compiler in an isolated directory with it's config file, and add the directory to the `PATH` when needed.